string stop_id
string departure_time
int32 seats_available
StopType[] remaining_stops 
